client/src/components/Instructions/index.js

```diff
@@ -1,103 +1,102 @@
 import React, { Component } from "react";
 import { Button } from "rimble-ui";
-import styles from './Instructions.module.scss';
+import styles from "./Instructions.module.scss";
 
 export default class Instructions extends Component {
-
   renderCounterSetup() {
     const addressDefault = this.getDefaultAddress();
     return (
       <div className={styles.instructions}>
         <h2> Build your first app with ZepKit </h2>
         <div className={styles.step}>
           <div className={styles.instruction}>
-            Add the Counter contract to your project.
+            Initialize your ZeppelinOS project
```

Comment:
I belive zos init should be done before running react app it is pointed out at https://zepkit.zeppelinos.org. We also ask them to do it readme file.

Reply:
Hmmm, yes I guess you're right.